README.md

```diff
@@ -53,16 +59,16 @@ the envfile:
 |---------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
 | `envkey_DEBUG`, `envkey_SOME_API_KEY` | These values can be whatever, and they will be added to the envfile as  `DEBUG`  and  `SOME_API_KEY` .                               |
 | `envkey_SECRET_KEY`                 | This one will use a secret stored in the repository's Github Secrets, and add it to the file as  `SECRET_KEY`                        |
-| `directory` (**Optional**)            | This key will set the directory in which you want to create `env` file. (Action will fail if the specified directory doesn't exist.) |
+| `directory` (**Optional**)            | This key will set the directory in which you want to create `env` file. **Important: cannot start with `/`** (Action will fail if the specified directory doesn't exist.) |
```

Comment:
```suggestion
| `directory` (**Optional**)            | This key will set the directory in which you want to create `env` file. **Important: cannot start with `/`. Action will fail if the specified directory doesn't exist.** |
```

I think it would be nice to format this section a little differently. Also, what do you mean it can't start with `/`? If you mean to say it is a relative path, then it might be worth saying that instead. Otherwise, maybe we should add some checks to check if the beginning of the path is `/` and handle an absolute path instead of a relative one. For now, I think we can just improve this line, and I'll create a separate issue to handle absolute paths sometime.

Reply:
I fix this